### PR TITLE
ORCA-784: Update/fix restore-to-orca.md document with current ORCA changes.

### PR DESCRIPTION
**Pull request by zroehrich**
## Summary of Changes

Addresses [ORCA-784: Update/fix restore-to-orca.md document with current ORCA changes.](https://bugs.earthdata.nasa.gov/browse/ORCA-784)

## Changes

* Changed documentation to replace restore with copy based on task's naming.

## Type of change

Please delete options that are not relevant.
- [x] This change requires a documentation update

## How Has This Been Tested?
Deployed website locally and verified that changes worked successfully.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the necessary documentation (remove those that do not apply)
    - [x] CHANGELOG.md
    - [x] Operator Documentation 
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
    - [x] Manual tests (outlined above)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ and includes an additional section for migration notes.
 
 - *ORCA-361* Removed hardcoded test values from `extract_file_paths_for_granule` unit tests.
 - *ORCA-710* Removing duplicate logging messages in `integration_test/workflow_tests/custom_logger.py`
+- *ORCA-784* Changed documentation to replace restore with copy based on task's naming.
```
Comment (rizbihassan) [inline CHANGELOG.md:28]: Add which specific documentation has been renamed.

**Review by rizbihassan — CHANGES_REQUESTED** [review]
The `website/docs/operator/restore-to-orca.mdx` should be renamed to website/docs/operator/reingest-to-orca.mdx` and website should be tested locally to verify it's not breaking anything.

File: CHANGELOG.md (hunk 1 above)
Comment (zroehrich) [reply CHANGELOG.md]: Updated with documentation path that has been renamed

Comment (zroehrich) [reply CHANGELOG.md]: Changed file name and updated sidebar.js. Verified locally that the changes did not break anything

File: website/docs/operator/reingest-to-orca.mdx [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-id: restore-to-orca
+id: reingest-to-orca
 title: Re-Ingesting Data to ORCA
 description: Provides documentation for Operators to restore data back to ORCA.
```
Comment (rizbihassan) [inline website/docs/operator/reingest-to-orca.mdx:4]: This line should also be updated with reingest.

Comment (zroehrich) [reply website/docs/operator/reingest-to-orca.mdx]: Good catch, updated

**Review by rizbihassan — APPROVED** [review]

**Pull request merged**